index.html

```diff
@@ -38,8 +38,8 @@ <h1 class="logo">Delicious</h1>
            </header>
            <section>
               <!--pizza image-->
-              <h1>Chocolate Pizza</h1>
-              <p>Posted on 15 Dec 2013 / Desserts</p>
+              <h1 class="chocoPizza" style="margin-bottom: 0px;">Chocolate Pizza</h1>
```

Comment:
Nice job using classes for style here.
Inline styles work, but it's better to just assign those properties through the class and css file so you don't have to repeat it on every page. Makes maintenance so much easier!